pnc_cli/user_config.py

```diff
@@ -70,25 +71,34 @@ def __setstate__(self, state):
         else:
             self.keycloak_config = saved_kc_config
 
+
         # if more than a day has passed since we saved the token, and keycloak server configuration is not modified,
         # get a new one
-        if not newtoken and utils.current_time_millis() - self.token_time > 86400000:
+        if not newtoken and utils.current_time_millis() - self.refresh_token_time > 8640000:
```

Comment:
Even though it probably doesn't help much, I would change `>` to `>=`

Reply:
Sure thing.